_progress-bar.scss

```diff
@@ -0,0 +1,252 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
```

Comment:
Using FontAwesome here raises a really interesting question around icons. Many components will need icons, and we don’t have a compelling story yet around what Stencil should do for its icon assets. For the moment, I’ll say I’m not crazy about using FontAwesome (or any particular icon set) as a requirement for using a Stencil component, but as noted, we don’t have a preferred approach worked out.

Don’t change this yet, but I’d like to revisit it after the Icon component is done and before releasing this officially.